portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,60 @@
 <!DOCTYPE html>
 <html>
   <head>
+    <style>
+body {
+  background-image: url('images/white-cat-1.png');
+}
+</style>
     <meta charset="UTF-8">
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+      <h1>The Portfolio</h1>
+      <b>(Scroll to the bottom for a surprise)</b>
+      <p>Hello, I am Anthony Achille, Check out my <a href="https://github.com/Anthony869?tab=repositories">GitHub</a></p>
+      <p></p>
+      <p></p>
+      <button onclick="addRandomGreeting()">Ricky Quotes</button>
       <div id="greeting-container"></div>
+      <p></p>
+      <p></p>
+      <strong>Background:</strong>  
+      <p>I have always been interested in programming and computers in general. Ever since I could remember I wnated to learn
+         more about computer and how they work. Technology has always been interesting and being able to control them with code.
+         As soon as I could take a class about computers in highschool I did, I took Computer Science 3 the first year and continued 
+         with the path after that. Before I could do that at school I went to camps over the summer to learn different languages each time.
+        </p>
+      <p>I also like to Golf and have for 9+ years. I have been playing since I was like 6 or 7 because my Nonno introduced me to it.
+         I've also played basketball and baseball but I didn't like them as much as I liked Golf. I tried out for the golf team the
+         first year I could when I was a freshman in highschool. The next year I did the Junior Varsity with a bunch of my friends.
+         I did the same thing this year with all my friends that golf for the very end of the season. 
+      </p>
+      <strong>Photography:</strong>
+      <p>I like to photogragh anything that I get the chance to take pictures of. Some of my favoritie pictures where the ones from
+         our trip to europe. I also like to take picutres of bird that are nearby since we have a lot of trees and a lot of possiblities
+         to see a bird. I also would like to take landscape photos but I haven't been there with my camera a lot of the time when I see
+         a landscape that is worth photograghing.
+      </p>
+      <img src="images/Micro, blue, harvest moon friday 13th.JPG" alt="Moon" style="width:450px;height:300px;">
```

Comment:
Note, in the future try to avoid spaces in filenames, and rename it to something with underscores instead.

They are harder to deal with in the linux shell, and even HTTP gives them special treatment in the URL. (ie. " " -> "%20")